kube-app-testing.sh

```diff
@@ -24,6 +24,13 @@ PRE_TEST_SCRIPT_PATH="ci/pre-test-hook.sh"
 DEFAULT_CLUSTER_TYPE=kind
 PYTHON_TESTS_DIR="test/kat"
 
+# gs cluster config
+DEFAULT_GS_RELEASE="11.2.1"
+GS_API_URI="https://api.g8s.gauss.eu-central-1.aws.gigantic.io"
+AVAILABILITY_ZONE="eu-central-1a"
+SCALING_MIN=1
+SCALING_MAX=2
```

Comment:
These will not stay hard-coded, it's just intended to keep things simple for the moment.